internal/controller/flow_controller_test.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2024.
```

Comment:
Please use:
https://github.com/Mellanox/spectrum-x-operator/blob/main/hack/boilerplate.go.txt

Reply:
will fix in the next PR.